### PR TITLE
feat(favicon): change default favicon, handle custom favicon

### PR DESCRIPTION
**Pull request by nogizhopaboroda**

### CODE REVIEW

**Comment by nogizhopaboroda** [comment]
@nvignola @pankajpatel new feature =)

File: server.js [hunk 1]
```diff
@@ -38,9 +39,15 @@ Promise.all([
   customFs.readFile(CUSTOM_PREVIEW_HEADER),
   getWebpackConfig(),
   customFs.checkFile(MIDDLEWARES_FILE),
-]).then(([header, webpackConfig, customMiddlewaresExist]) => {
+  customFs.checkFile(FAVICON_FILE),
+]).then(([header, webpackConfig, customMiddlewaresExist, faviconFileExists]) => {
   const compiler = webpack(webpackConfig);
 
+  if(faviconFileExists){
+    console.log('using custon favicon');
```
Comment (pankajpatel) [inline server.js:47]: typo? do we need this comment and if case?

**Review by nvignola — APPROVED** [review]
Cool!

**Review by pankajpatel — APPROVED** [review]

File: server.js (hunk 1 above)
Comment (nogizhopaboroda) [reply server.js:47]: no typo. just info along with https://github.com/modulor-js/modulor-storybook/blob/master/config.js#L19

Comment (pankajpatel) [reply server.js:47]: alright!

**Pull request merged**